src/rwe/MainMenuScene.cpp

```diff
@@ -725,18 +726,22 @@ namespace rwe
 
     std::vector<std::string> MainMenuScene::getMapNames()
     {
-        auto mapNames = sceneContext.vfs->getFileNames("maps", ".ota");
-
-        for (auto& e : mapNames)
+        if (cachedMapNames.empty())
```

Comment:
It would be nice to distinguish between the user genuinely having no maps and the cache not being populated yet.